services/src/floodlight-modules/src/main/resources/floodlightkilda.properties

```diff
@@ -48,7 +48,7 @@ net.floodlightcontroller.core.internal.OFSwitchManager.clearTablesOnEachTransiti
 net.floodlightcontroller.core.internal.OFSwitchManager.keyStorePath=/path/to.jecks
 net.floodlightcontroller.core.internal.OFSwitchManager.keyStorePassword=PassFL
 net.floodlightcontroller.core.internal.OFSwitchManager.useSsl=NO
-net.floodlightcontroller.core.internal.OFSwitchManager.supportedOpenFlowVersions=1.0, 1.1, 1.2, 1.3, 1.4, 1.5
+net.floodlightcontroller.core.internal.OFSwitchManager.supportedOpenFlowVersions=1.2, 1.3
```

Comment:
This change are aftermath from not synced "templates"... as far as they are in template, they must be into local config too.

Because I have make a sync with template, this changes come here.

From my point of view it is not a big crime to merge one config update with changes that already must be in this config.

Separate PR will raise too many questions... But in future I will try to split such changes.

Reply:
Just put the first sentence from your comment to PR description, and no questions like this will appear.